src/component/Home.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
```

Comment:
- [ ] Good job so far 👍 Kindly install all the dependencies and make sure they are present in your `package.json` file. Right now your react app does not work since `npm i` cannot find those dependencies to install. Kindly rectify this issue 💯 
- Console error:
![Screenshot 2022-07-13 060748](https://user-images.githubusercontent.com/85113567/178655836-6bd9c4dc-1fe3-4836-a1d0-e8faa1bad254.png)